### PR TITLE
Pagination for person

### PR DESCRIPTION
**Pull request by gadavid**

### CODE REVIEW

File: web/templates/person/show.html.eex [hunk 1]
```diff
@@ -7,7 +7,16 @@
     <input id='show_events' type='checkbox'> Show joins + parts
   </label>
 </p>
-
+<p>
+  <%= if @page_number > 1 do %>
+  <a href="<%= person_path @conn, :show, "gadavid" %>/<%= @page_number-1 %>">< Prev</a>
```
Comment (radar) [inline web/templates/person/show.html.eex]: I would use `&laquo;` here before previous. The < might be interpreted as a HTML tag by browsers.

File: web/templates/person/show.html.eex [hunk 2]
```diff
@@ -7,7 +7,16 @@
...
+  |
+  <% end %>
+  Page <%= @page_number %> of <%= @total_pages %>
+  <%= if @page_number < @total_pages do %>
+  |
+  <a href="<%= person_path @conn, :show, "gadavid" %>/<%= @page_number+1 %>"> Next ></a>
```
Comment (radar) [inline web/templates/person/show.html.eex]: Same as above, I would use `&raquo;`

**Comment by gadavid** [comment]
Implemented the changes

**Comment by radar** [comment]
Thank you @gadavid!

**Pull request merged**